commands/clean.js

```diff
@@ -1,39 +1,37 @@
-const { execSync } = require('child_process')
-const ProcessManager = require('pm2')
-const { existsSync } = require('fs')
-const { homedir } = require('os')
-const path = require('path')
+const { execSync } = require('child_process');
+const ProcessManager = require('pm2');
+const { existsSync } = require('fs');
+const { homedir } = require('os');
+const path = require('path');
```

Comment:
style in this repo is without `;`. And eslint is depreciated (currently we rely on prettier), can you update the formatting?

Reply:
why no ;, seems unholy to me
will remove it